tests/e2e/helm-chart/02-install-simple-demo.yaml

```diff
@@ -1,31 +1,37 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: membership
+  name: coupon
```

Comment:
I see most of the yaml step files, like this, Are the same at helm-chart and multi-apps? 
Why do we need them both?

Reply:
I think we can do some "Common" folder where all the same yaml files will be there.

Reply:
I plan to change one of the scenarios to install a different app (probably [kv-mall](https://github.com/keyval-dev/kv-mall)). So they are going to be different (will happen in a future PR)